packages/calcite-components/src/components/meter/meter.scss

```diff
@@ -0,0 +1,117 @@
+@include base-component();
+
+:host {
+  @apply flex;
+  --calcite-meter-space-internal: theme("spacing[0.5]");
+  --calcite-meter-height-internal: theme("spacing.4");
+  --calcite-meter-font-size-internal: var(--calcite-font-size--1);
+}
+
+:host([scale="s"]) {
+  --calcite-meter-height-internal: theme("spacing.3");
+  --calcite-meter-font-size-internal: var(--calcite-font-size--2);
+}
+
+:host([scale="l"]) {
+  --calcite-meter-height-internal: theme("spacing.6");
+  --calcite-meter-font-size-internal: var(--calcite-font-size-0);
+}
+
+.meter {
+  @apply flex relative items-center w-full m-0;
+  block-size: var(--calcite-meter-height-internal);
+  background-color: var(--calcite-ui-foreground-2);
+  border: 1px solid var(--calcite-ui-border-3);
+  border-radius: var(--calcite-meter-height-internal);
+}
+
+.meter.solid {
+  border: 1px solid var(--calcite-ui-foreground-3);
+  background-color: var(--calcite-ui-foreground-3);
+}
+
+.meter.outline {
+  @apply bg-transparent;
+}
+
+.meter-value-visible {
+  margin-block-start: theme("spacing.6");
+}
+
+.meter-steps-visible {
+  margin-block-end: theme("spacing.6");
+}
+
+.meter-step-line {
+  @apply block absolute inset-y-0;
+  inline-size: var(--calcite-meter-space-internal);
+  background-color: var(--calcite-ui-border-3);
+}
+
+.meter-label {
+  @apply absolute;
+  font-size: var(--calcite-meter-font-size-internal);
+}
+
+.meter-label-value {
+  inset-block-end: calc(100% + 0.5em);
+  font-weight: var(--calcite-font-weight-bold);
+  color: var(--calcite-ui-text-1);
+}
+
+.meter-label-range {
+  inset-block-start: calc(100% + 0.5em);
+  color: var(--calcite-ui-text-3);
+}
+
+.meter-unit-label {
+  font-weight: var(--calcite-font-weight-medium);
+  color: var(--calcite-ui-text-3);
+}
+
+.meter-label-value .meter-unit-label {
+  color: var(--calcite-ui-text-2);
+  font-weight: var(--calcite-font-weight-bold);
+}
+
+.meter-fill {
+  @apply block absolute duration-150 ease-in-out bg-brand z-sticky;
```

Comment:
Should this use the default z-index util instead? The sticky level seems too high, IMO.

Reply:
Don't think we need this at all - removing.